library/kotlin/io/envoyproxy/envoymobile/EngineBuilder.kt

```diff
@@ -606,6 +591,15 @@ open class EngineBuilder(
     return this
   }
 
+  /**
+   * Injects additional stat sink configurations. This should be a YAML/JSON formatted extension
```

Comment:
Make this doc string consistent with the Swift one (which currently has more information)